syft/__init__.py

```diff
@@ -140,3 +140,6 @@ def pool():
     from syft.generic.id_provider import IdProvider
 
     ID_PROVIDER = IdProvider()
+
+# Garbage colect all remote data on a worker every garbage_delay seconds
+garbage_delay = 0
```

Comment:
Q: Wouldn't this be a better attribute for the ```storage``` class?

Reply:
👍